package.json

```diff
@@ -41,7 +41,8 @@
     "webpack-cli": "^5.1.1"
   },
   "dependencies": {
-    "@stellar/stellar-sdk": "^11.1.0",
+    "@stellar/freighter-api": "^1.7.1",
+    "@stellar/stellar-sdk": "stellar/js-stellar-sdk#bindings",
```

Comment:
@piyalbasu just out of curiosity, what's the reason behind this "#bindings" change?

Reply:
This was targeted at a branch of stellar-sdk that had the TS bindings update for dev purposes. This work may be merged already. I'll follow up